lldb/source/Plugins/SymbolLocator/Debuginfod/SymbolLocatorDebuginfodProperties.td

```diff
@@ -0,0 +1,8 @@
+include "../../../../include/lldb/Core/PropertiesBase.td"
+
+let Definition = "symbollocatordebuginfod" in {
+  def URLs: Property<"urls", "String">,
+    Global,
+    DefaultStringValue<"">,
+    Desc<"A space-separated, ordered list of Debuginfod server URLs to query for symbols">;
+}
```

Comment:
What does this look like on the command line? Other plug-in settings look like this:
```
plugin.dynamic-loader.darwin-kernel.load-kexts (boolean) = true
plugin.dynamic-loader.darwin-kernel.scan-type (enum) = fast-scan
plugin.jit-loader.gdb.enable (enum) = default
plugin.object-file.pe-coff.abi (enum) = default
plugin.object-file.pe-coff.module-abi (dictionary of enums) =
plugin.process.kdp-remote.packet-timeout (unsigned) = 5
plugin.process.gdb-remote.packet-timeout (unsigned) = 5
plugin.process.gdb-remote.target-definition-file (file) = 
plugin.process.gdb-remote.use-g-packet-for-reading (boolean) = false
plugin.process.gdb-remote.use-libraries-svr4 (boolean) = true
plugin.symbol-file.dwarf.ignore-file-indexes (boolean) = false
plugin.structured-data.darwin-log.auto-enable-options (string) = 
plugin.structured-data.darwin-log.enable-on-startup (boolean) = false
```
So hopefully this show up in `settings set plugin.symbolvendor.symbollocatordebuginfod.urls <url1> [<url2>]`

Reply:
It looks like this now:
```
...
plugin.symbol-file.dwarf.ignore-file-indexes (boolean) = false
plugin.symbol-locator.debuginfod.server_urls (array of strings) =
  [0]: "https://debuginfod.centos.org/"
plugin.structured-data.darwin-log.auto-enable-options (string) =
...
```